ObservatoryFramework/Files/Converters/MutableStringDoubleConverter.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Immutable;
+using System.Text;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace Observatory.Framework.Files.Converters
+{
+    class MutableStringDoubleConverter : JsonConverter<object>
+    {
+        public override object Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
+        {
+            if (reader.TokenType == JsonTokenType.String)
```

Comment:
Here and 2x in Modifiers.cs: Consider a ternary (`?:`) operator.

Reply:
I _do_ like ternaries...